packages/bot/src/scratch/hooks/toolbox.js

```diff
@@ -198,18 +198,24 @@ Blockly.Toolbox.prototype.showCategory_ = function (category_id) {
         flyout_content.blocks = flyout_content.blocks.concat(uniqueProceBlocks);
     } else {
         const selected_category = this.categoryMenu_.categories_.find(category => category.id_ === category_id);
-        flyout_content = selected_category.getContents();
-
-        // Dynamic categories
-        if (typeof flyout_content === 'string') {
-            const fnToApply = this.workspace_.getToolboxCategoryCallback(flyout_content);
-            flyout_content = fnToApply(this.workspace_);
-        }
+        flyout_content = this.getCategoryContents(selected_category, this.workspace_);
```

Comment:
Since this function is part of the prototype there is no need to pass `this.workspace_`, you can just reference it in the function itself.